devel/cmake/files/patch-CMakeFindFrameworks.cmake.diff

```diff
@@ -1,10 +1,11 @@
 --- Modules/CMakeFindFrameworks.cmake.orig
 +++ Modules/CMakeFindFrameworks.cmake
-@@ -19,6 +19,7 @@
-     if(APPLE)
-       foreach(dir
-           ~/Library/Frameworks/${fwk}.framework
-+          __PREFIX__/Library/Frameworks/${fwk}.framework
-           /usr/local/Frameworks/${fwk}.framework
-           /Library/Frameworks/${fwk}.framework
-           /System/Library/Frameworks/${fwk}.framework
+@@ -22,7 +22,7 @@
+             ${CMAKE_FRAMEWORK_PATH}
+             ${_cmff_CMAKE_FRAMEWORK_PATH}
+             ~/Library/Frameworks
+-            /usr/local/Frameworks
++            __PREFIX__/Frameworks
```

Comment:
This should be `__FRAMEWORKS_DIR__` rather than `__PREFIX__/Frameworks`. If you want to stay consistent with how this was before, you should add that between the `~/Library/Frameworks` and `/usr/local/Frameworks` lines, rather than replacing the `/usr/local/Frameworks` line. The reinplace will also have to be adjusted to read:
```
        reinplace "s|__PREFIX__|${prefix}|g" \
            ${worksrcpath}/macports.cmake \
            ${worksrcpath}/Modules/Platform/Darwin.cmake
        reinplace "s|__FRAMEWORKS_DIR__|${frameworks_dir}|g" \
            ${worksrcpath}/Modules/CMakeFindFrameworks.cmake
```

Reply:
Good point! I'll update.